### ISSUE TITLE
[private-cluster-update-variant] Module does not allow to update node pool labels without pool recration

### ISSUE THREAD

**Issue by alarex** [terraform-google-modules/terraform-google-kubernetes-engine#1695]
### TL;DR

Modules [private-cluster-update-variant](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/c4809044b52b91505bfba5ef9f25526aa0361788/modules/private-cluster-update-variant/cluster.tf#L419) and [beta-private-cluster-update-variant](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/v27.0.0/modules/beta-private-cluster-update-variant/cluster.tf#L488) still forces node pools to be recreated, while since [4.48.0](https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#4480-january-9-2023) labels [allowed](https://github.com/hashicorp/terraform-provider-google-beta/pull/4998/files) to be updated in-place.

Can it be fixed by just removing labels block from keepers?

### Expected behavior

Updating lables for node pool updates it in-place.

### Observed behavior

Updating labels in module forces random_id to recreate and it leads to node_pool be recreated too.

### Terraform Configuration

```hcl
module "gke-usc1" {
  source     = "terraform-google-modules/kubernetes-engine/google//modules/beta-private-cluster-update-variant"
  version    = "27.0.0"
### Non-important stuff
  node_pools = [
    {
      name                        = "g2-std-4"
      accelerator_count           = 1
      accelerator_type            = "nvidia-l4"
      auto_repair                 = true
      auto_upgrade                = true
      autoscaling                 = true
      cpu_cfs_quota               = true
      disk_size_gb                = 60
      disk_type                   = "pd-ssd"
      enable_gcfs                 = false
      enable_gvnic                = false
      enable_integrity_monitoring = true
      enable_secure_boot          = true
      image_type                  = "COS_CONTAINERD"
      initial_node_count          = 0
      local_ssd_count             = 0
      local_ssd_ephemeral_count   = 0
      location_policy             = "BALANCED"
      machine_type                = "g2-standard-4"
      max_count                   = 1024
      max_pods_per_node           = 16
      max_surge                   = 3
      max_unavailable             = 1
      min_count                   = 0
      node_locations              = "us-central1-a,us-central1-b"
    },
  ]
  node_pools_labels = {
    all = {}
    g2-std-4 = {
      "example_label" = "true",
    }
  }
}
```

### Terraform Version

```sh
1.5.0
```

**Comment by lauraseidler** [comment]
We're running into the same issue - I'll test if removing the keeper is enough, and will open a PR

**Comment by lauraseidler** [comment]
Not stale, this is still an open issue
